ca/django_ca/models.py

```diff
@@ -497,7 +492,8 @@ class CertificateAuthority(X509CertMixin):
     parent = models.ForeignKey(
         "self", on_delete=models.SET_NULL, null=True, blank=True, related_name="children"
     )
-    private_key_path = models.CharField(max_length=256, help_text=_("Path to the private key."))
+    key_backend_path = models.CharField(max_length=256, help_text=_("Backend to handle private keys."))
+    key_backend_options = models.JSONField(default=dict, help_text=_("Key backend options"))
```

Comment:
`key_backend_path` is a Python class path and should represent a type implementing a certain interface.
 `key_backend_options` is the options to pass to an instance of that interface.